.travis.yml

```diff
@@ -51,3 +51,4 @@ addons:
     branch_pattern: coverity-scan
 
 script: cd src && make && make test
+script: ./kak -e "buffer *debug*; write debug_buffer\n; quit" && echo "*** This is the debug buffer, where debug info will be written ***\n" | cmp debug_buffer -
```

Comment:
This line is also depending on some string in the source code that might change or be removed in the future, I don't think this is the best way of testing whether all scripts have been loaded.

Also when the test suite is run, the contents of the debug buffer will be displayed, which will show that some scripts haven't been loaded, so we might not need to run this pipeline at all.